### PR TITLE
base-files: banner: Replace OpenWRT banner with a Freifunk banner

### PR DESCRIPTION
**Pull request by PolynomialDivision**
base-files: banner: replace openwrt banner with freifunk

Based on [Berlin Luci Banner](https://github.com/freifunk-berlin/firmware/blob/master/patches/700-banner-info.patch).

### CODE REVIEW

File: patches/openwrt/0900-replace-banner-with-freifunk.patch [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+- |       |.-----.-----.-----.|  |  |  |.----.|  |_
+- |   -   ||  _  |  -__|     ||  |  |  ||   _||   _|
+- |_______||   __|_____|__|__||________||__|  |____|
+-          |__| W I R E L E S S   F R E E D O M
++  _____        _  __             _
++ |  ___|      (_)/ _|           | |
++ | |_ _ __ ___ _| |_ _   _ _ __ | | __
++ |  _| '__/ _ \ |  _| | | | '_ \| |/ /
++ | | | | |  __/ | | | |_| | | | |   <
++ \_| |_|  \___|_|_|  \__,_|_| |_|_|\_\
```
Comment (mweinelt) [inline patches/openwrt/0900-replace-banner-with-freifunk.patch]: > Gluon is a modular framework for creating OpenWrt-based firmwares for wireless mesh nodes. Several Freifunk communities in Germany use Gluon as the foundation of their Freifunk firmwares.

While several Freifunk communities use Gluon to build their firmwares, it is not a dedicated Freifunk firmware framework.

File: patches/openwrt/0900-replace-banner-with-freifunk.patch [hunk 2]
```diff
@@ -15,12 +15,12 @@ index 799f7e5..628447c 100644
...
-+ | |_ _ __ ___ _| |_ _   _ _ __ | | __
-+ |  _| '__/ _ \ |  _| | | | '_ \| |/ /
-+ | | | | |  __/ | | | |_| | | | |   <
-+ \_| |_|  \___|_|_|  \__,_|_| |_|_|\_\
++   _____ _     _    _  ____  _   _ 
++  / ____| |   | |  | |/ __ \| \ | |
++ | |  __| |   | |  | | |  | |  \| |
++ | | |_ | |   | |  | | |  | | . ` |
++ | |__| | |___| |__| | |__| | |\  |
++  \_____|______\____/ \____/|_| \_|
```
Comment (PolynomialDivision) [inline patches/openwrt/0900-replace-banner-with-freifunk.patch]: @mweinelt 
Maybe like this? But there is no need to merge this pull request. ^^

File: patches/openwrt/0900-replace-banner-with-freifunk.patch [hunk 3]
```diff
@@ -29,8 +29,8 @@ index 799f7e5..628447c 100644
 -  * 3/4 oz Lime Juice       unstrained into a goblet.
 -  * 1 1/2 oz Orange Juice
 -  * 1 tsp. Grenadine Syrup
-+ Gluon Firmware (%N %V rev %C)
-+ %P - %S
++ Gluon Firmware (Chaos Calmer 15.05.1 rev Chaos Calmer)
++ Generic - x86/64
 + https://github.com/freifunk-gluon/gluon
 + https://freifunk.net/
```
Comment (PolynomialDivision) [inline patches/openwrt/0900-replace-banner-with-freifunk.patch]: This should be deleted too...

**Comment by NeoRaider** [comment]
We generally don't accept non-upstreamable patches in Gluon. We have a few old ones that will need considerable rework before being upstreamed (in total, Gluon contains about 10 patches that aren't upstream at the moment), but we'd rather like do decrease that number than add new ones.

**Comment by rotanid** [comment]
there are custom packages out there that you can use to replace the default banner

**Comment by PolynomialDivision** [comment]
@rotanid : Can you give me the name of this package? :)

**Comment by mweinelt** [comment]
`gluon-banner` 
https://github.com/freifunk-ffm/packages/tree/master/ffffm-banner

**Comment by PolynomialDivision** [comment]
Yeah but I don't like this solution... You can't add the placeholders and so on...
So no information about version and stuff like this...

I'm doing something like this now:
https://github.com/PolynomialDivision/site/commit/bf8434eaeee9e33f41eb6163b585cfec2f3a5bf5

**Pull request closed without merging**